docs/update.js

```diff
@@ -69,74 +107,79 @@ function createToggles(map) {
+    povertyWeight.appendChild(povertyWeightValue);
+    weightInputDiv.appendChild(povertyWeight);
 
-    form.appendChild(thresholdInputDiv);
-  }
+    weightInputDiv.appendChild(document.createElement('br'));
+
+    const weightInput = createBasicToggleInputElement('poverty weight');
+    weightInput.type = 'range';
+    weightInput.min = '0.00';
+    weightInput.max = '1.00';
+    weightInput.oninput = updateWeights;
+    weightInputDiv.appendChild(weightInput);
 
-  // weight toggle
-  const weightInputDiv = document.createElement('div');
-  weightInputDiv.className = 'input-container';
-  const weightTitle = document.createElement('div');
-  weightTitle.className = 'formTitle';
-  weightTitle.innerHTML = 'weights';
-  weightInputDiv.appendChild(weightTitle);
-
-  const povertyWeight = document.createElement('label');
-  povertyWeight.innerHTML = 'poverty weight: ';
-  const povertyWeightValue = document.createElement('span');
-  povertyWeightValue.id = povertyWeightValueId;
-  povertyWeight.appendChild(povertyWeightValue);
-  weightInputDiv.appendChild(povertyWeight);
-
-  weightInputDiv.appendChild(document.createElement('br'));
-
-  const weightInput = createBasicToggleInputElement('poverty weight');
-  weightInput.type = 'range';
-  weightInput.min = '0.00';
-  weightInput.max = '1.00';
-  weightInput.oninput = updateWeights;
-  weightInputDiv.appendChild(weightInput);
-
-  weightInputDiv.appendChild(document.createElement('br'));
-
-  const damageWeight = document.createElement('label');
-  damageWeight.innerHTML = 'damage weight: ';
-  const damageWeightValue = document.createElement('span');
-  damageWeightValue.id = damageWeightValueId;
-  damageWeight.appendChild(damageWeightValue);
-  weightInputDiv.appendChild(damageWeight);
-
-  form.appendChild(weightInputDiv);
+    weightInputDiv.appendChild(document.createElement('br'));
+
+    const damageWeight = document.createElement('label');
+    damageWeight.innerHTML = 'damage weight: ';
+    const damageWeightValue = document.createElement('span');
+    damageWeightValue.id = damageWeightValueId;
+    damageWeight.appendChild(damageWeightValue);
+    weightInputDiv.appendChild(damageWeight);
+
+    form.appendChild(weightInputDiv);
+  }
 
   // buttons
-  form.appendChild(createButton('update', () => {
-    update(map);
-  }));
+  form.appendChild(createButton('update', () => update(map)));
   form.appendChild(document.createElement('br'));
   form.appendChild(createButton('current settings', reset));
 
   document.getElementById('form-div').appendChild(form);
   updateWeights();
 }
 
+/**
+ * Creates an div including input and label for the given toggle.
+ * @param {string} toggle
+ * @return {HTMLDivElement}
+ */
+function createInput(toggle) {
+  const thresholdInputDiv = document.createElement('div');
+  thresholdInputDiv.className = 'input-container';
+
+  const label = document.createElement('label');
+  label.for = toggle;
+  label.id = 'for-' + toggle;
```

Comment:
why do labels need ids?

Reply:
Yeah, this whole file is kind of a mess (re: this comment and below). Wrote it in the early days. I just functionalized this code without changing it (I believe). I originally started to make it better but felt like it was obscuring all the actual changes. I can do the updates here if you're fine with that or add a TODO.  Removed the one label id for now but I think there's a bit more to be done here.

Reply:
Go as far as you want: github can't cope with indentation diffs anyway, so it's as if it were all new